core/lib/dal/src/models/storage_eth_tx.rs

```diff
@@ -22,6 +22,7 @@ pub struct StorageEthTx {
     pub updated_at: NaiveDateTime,
     // TODO (SMA-1614): remove the field
     pub sent_at_block: Option<i32>,
+    pub blob_sidecar: Option<Vec<u8>>,
```

Comment:
Add a comment and explain how this data is generated (and that even if we have multiple blobs, they are still within a single sidecar)